betelgeuse/parser.py

```diff
@@ -187,3 +187,31 @@ def parse_docstring(docstring=None):
             field_value = output
             fields_dict[field_name] = field_value
     return fields_dict
+
+
+def parse_markers(all_markers=None, config=None):
+    """Parse the markers."""
+    resolved_markers = []
+    ignore_list = getattr(config, 'MARKERS_IGNORE_LIST', None)
+
+    def _process_marker(marker):
+        # Fetching exact marker name
+        marker_name = marker.split('mark.')[-1] if 'mark' in marker else marker
+
+        # ignoring the marker if in ignore list
+        if ignore_list and not any(
```

Comment:
```suggestion
        if not any(
```
only accept if you accepted the default value proposal

Reply:
Nope it works differently, checking if the ignore_list is present or not is important:

Here is the example, in case of ignore_list is empty:

```
In [7]: ignore_list = []

In [11]: if not any(ignore_word in 'match3' for ignore_word in ignore_list):
    ...:     print('Y')
    ...: 
Y

In [12]: if ignore_list and not any(ignore_word in 'match3' for ignore_word in ignore_list):
    ...:     print('Y')
    ...: 
```
Notice `In [12]:` does not prints `Y` !

Reply:
In simple words, python checks for none in string and returns True 🤣 

Also, this would skip the code execution for those who dont want use this feature of ignored markers altogether fastening the process a bit but not huge!

Reply:
I don't get your point. I thought you actually want to append the marker if it does not match anything in the `ignore_list`. Or what am I missing?

Reply:
Corrected after discussing on Slack! Nice catch !!